packages/neo-one-website/docs/0-installation/2-playground.md

```diff
@@ -43,6 +45,7 @@ This will start up a local network, compile the smart contracts located in the `
 
 ```bash
 yarn start
+# npm run ts-node ./scripts/start.ts
```

Comment:
`npm run start` doesn't work here? If not, then it should probably be `npm run-script start`

Reply:
I'm pretty sure I tried that and it failed with some errors.  I just verified that "npm run start" works.

Reply:
👍